operator/images/access-setup/content/bin/setup_work_dir.sh

```diff
@@ -174,20 +179,42 @@ tekton_results_manifest(){
   results_kustomize="$manifests_dir/compute/tekton-results/kustomization.yaml"
   results_namespace="$manifests_dir/compute/tekton-results/namespace.yaml"
   results_secret="$manifests_dir/compute/tekton-results/tekton-results-secret.yaml"
+  results_minio_secret="$manifests_dir/compute/tekton-results/tekton-results-minio-secret.yaml"
   if [ ! -e "$results_kustomize" ]; then
     results_dir="$(dirname "$results_kustomize")"
     mkdir -p "$results_dir"
     if [[ -z $TEKTON_RESULTS_DATABASE_USER || -z $TEKTON_RESULTS_DATABASE_PASSWORD ]]; then
       printf "[ERROR] Tekton results database variable is not set, either set the variables using \n \
       the config.yaml under tekton_results_db \n \
+      Or create '%s' \n" "$results_minio_secret" >&2
+      exit 1
+    fi
+
+    if [[ -z $TEKTON_RESULTS_MINIO_USER || -z $TEKTON_RESULTS_MINIO_PASSWORD ]]; then
+      printf "[ERROR] Tekton results log variable is not set, either set the variables using \n \
+      the config.yaml under tekton_results_log \n \
       Or create '%s' \n" "$results_secret" >&2
       exit 1
     fi
 
     kubectl create namespace tekton-results --dry-run=client -o yaml > "$results_namespace"
     kubectl create secret generic -n tekton-results tekton-results-database --from-literal=DATABASE_USER="$TEKTON_RESULTS_DATABASE_USER" --from-literal=DATABASE_PASSWORD="$TEKTON_RESULTS_DATABASE_PASSWORD" --dry-run=client -o yaml > "$results_secret"
 
-    yq e -n '.resources += ["namespace.yaml", "tekton-results-secret.yaml"]' > "$results_kustomize"
+    echo "---
```

Comment:
We avoid inlining yaml files as it prevents linting. Have a template that you modify on the fly, preferably using `yq`.